test/e2e/testdata/crd-creation/crd.yaml

```diff
@@ -39,6 +39,8 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   name: clusterdummies.argoproj.io
+  labels:
+    e2e.argoproj.io: "true"
```

Comment:
The absence of this label was allowing the duplicated-resources test to pass when it shouldn't have. Without the label, the e2e suite doesn't clean up the CRD. So it was already applied when the duplicated-resources test started.